src/app.md

```diff
@@ -725,6 +726,50 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-latest]: #ex-semver-latest
+<a name="ex-semver-latest"></a>
+## Find the latest version satisfying given range
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Given an array of unparsed version `&str`s, iterate and filter against a semver `VersionReq` and find the latest `Version`.
+
+```rust
+#[macro_use]
+extern crate error_chain;
+extern crate semver;
+
+use semver::Version;
+use semver::VersionReq;
+
+mod errors {
+    error_chain! { }
+}
+
+error_chain! {
+    foreign_links {
+        SemVer(semver::SemVerError);
+    }
+}
+
+fn run() -> Result<()> {
+    let vset = ["0.9.0", "1.0.0", "1.0.1"];
+    let r = VersionReq::parse("<= 1.0.0")
+        .chain_err(|| "unable to parse VersionReq")?;
```

Comment:
you can drop the `chain_err` if you add `semver::ReqParseError` to the `foreign_links`